about.html

```diff
@@ -22,13 +24,19 @@
 That includes behaviour from other instances - we have no problem with stopping to federate with asshole instances, or suspend contact with single accounts.
 </p>
 
-<hr>
+<h2>Publication rules</h2>
+
+We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance and the Fediverse in general a place of dialogue and interaction. Therefore we established some rules. Please be aware, that any repeated disregard of these rules will lead to silencing or suspension of your account.
+
+<h3>Bots</h3>
+
+Configure all automated posts, like bots, to be <strong>unlisted</strong>. This means those posts can be seen by everybody but do not show up on any <a href="https://chaos.social/web/timelines/public/local">local</a> or <a href="https://chaos.social/web/timelines/public">federated timeline</a>.
+
+<h3>Third party content</h3>
```

Comment:
I'd remove this sub-heading and replace "Bots" with "Bots and automated posts", since the issue is not with "third-party content", which is a bad term for what's going on here.

Reply:
I see this as a crucial difference: third party content, which gets added, is in my opinion 
something very different than bots. For example one of the main differences are the motivation why you don't want it on your platform.

Bots: because they are automated and not humans.
Third-party content: because you don't want half of your local timeline to show content coming from Twitter (but they are humans).

This was the main confusion I had, when I read the sentence as it is currently. Because I was not expecting that content from Twitter would be seen as bad as bots, and the argumentation was completely mixed up and not understandable - at least for me.

Reply:
Sorry, I'm new to Mastodon - and I just discovered my deepest fault and confusion. @rixx explained to me, that also native boosts on Mastodon are generally handled as unlisted. So why should Twitter then be something better. Then, of course, it makes completely sense what you are saying @rixx, but for the people coming from Twitter I suggest to explain it briefly.

Reply:
I don't quite agree – our about/more page is on the long side already. The longer the page, the less people will actually read it. Explaining how Mastodon works is not the job of the about page.

Reply:
I'd also say that subheadings on h3 level make the page harder to read, and simply putting <strong> tags around the relevant words of a paragraph would work better.

Reply:
So, I gave it a try. Now it would be understandable for a newbie like me. What do you think?

Reply:
Removed `<h3>` level headings.